tox.ini

```diff
@@ -21,15 +21,14 @@ commands = coverage combine
 [testenv:style]
 skip_install = True
 deps = ruff
-commands = ruff check --diff
-           ruff check --output-format=full
+commands = ruff check
            ruff format --diff
 
 [testenv:reformat]
 skip_install = True
 deps = ruff
-commands = ruff check --fix {posargs:}
-           ruff format {posargs:}
+commands = ruff check --fix
```

Comment:
The `{posargs:}` was overkill, and wrong a lot of the time since the 2 commands don't have identical switches.

One is better off simply installing `ruff` standalone (`pickley install ruff`), and then run `ruff check ...` directly.